src/App.css

```diff
@@ -84,20 +95,31 @@ a:visited {
 
 .delete-button {
   position: absolute;
-  top: 5vh;
-  right: 5vh;
-  font-size: 1rem;
+  top: 7%;
+  right: 6%;
+  font-size: 0.8rem;
+  opacity: 90%;
```

Comment:
This should be `0.9` instead.

Reply:
If I make that .9, then the button shrinks on hover. I am using the font-size as the transition so that it moves from .8 to .85 on hover.